tutorials/template.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: <!--- Give your tutorial a meaningful title starting with an imperative, for example: \"Build Your First QA System\" --->\n",
+    "\n",
+    "- **Level**: <!--- Who is this tutorial for? Choose one of the options: Beginner, Intermediate, Advanced --->\n",
+    "- **Time to complete**: <!--- Enter the time it takes to complete this tutorial, in minutes --->\n",
+    "- **Prerequisites**: <!--- If there's anything the users need to know or prepare, list it here. It can be \"the knowledge of basic NLP concepts\" or installing some prerequisite software. --->\n",
+    "- **Nodes Used**: <!--- List the nodes used in this tutorial, for example: `FARMReader`--->\n",
+    "- **Goal**: After completing this tutorial, you will have achieved/learned about <!--- here state the goal of the tutorial. What will the users have done when they complete it? --->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "<!--- Here provide a short description of the tutorial. What does it teach? What's its expected outcome? --->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 1\n",
+    "\n",
+    "<!--- Divide your tutorial into tasks. Each task is a separate section. The title of the sections should start with a verb in the -ing form, for example: Installing Haystack, Preparing the Collab Environment, and so on. --->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add some code to run, if applicable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Task 2"
```

Comment:
Why don't we show them a numbered list to encourage more step by step instructions?
1.
2.
3.